ClientCLI/ClientCLI.cpp

```diff
@@ -3,8 +3,26 @@
 
 #include <iostream>
 #include "..\Client\Client.h"
+#include "..\Communication\Communication.h"
```

Comment:
There is files like `...\Communication\Create.h" which has the Create namespace which we can use for things like
`Create::ClientRequest()`